pkg/constants/constants.go

```diff
@@ -20,6 +20,11 @@ const (
 	AdsMode      = "ads"
 	WorkloadMode = "workload"
 
+	// DataPlaneModeLabel is the label used to indicate the data plane mode
+	DataPlaneModeLabel = "istio.io/dataplane-mode"
+	// DataPlaneModeKmesh is the value of the label to indicate the data plane mode is kmesh
+	DataPlaneModeKmesh = "kmesh"
```

Comment:
It seems that the letter case is not same with before；Kmesh->kmesh

Reply:
good catch, we did treat both `Kmesh` and `kmesh` valid with case-insensitive comparing